dotcom-rendering/src/web/browser/sentryLoader/sentry.ts

```diff
@@ -50,11 +51,6 @@ Sentry.init({
 	},
 });
 
-Sentry.configureScope((scope) => {
-	scope.setTag('edition', editionLongForm);
-	scope.setTag('contentType', contentType);
-});
```

Comment:
Removed in favour of simplicity.

It made sense to include this extra data on errors when it was freely available but now we'd need to refactor the window object to also include these properties which feels excessive